Added Hover Transition and Enhanced search bar

ohmschrodinger opened the pull request:
#7 

@lord-cyclone100 please let me know if any adjustments are needed 😄

lord-cyclone100 commented:
gr8 work!!!
But I suggest some changes.....
+ for the pokemon card, apply the hover and shadow effect for the pokemon image and not the card......use drop shadow instead of box shadow so that the shadow effect is applied wrt to the pokemon outlines.

+ Also it is not advised to use pure black and pure white colors (#000 and #fff) in any website. I saw somewhere in css u gave pure white color. pls revert back to the original one.

Make these changes then I shall merge. Also look at the comments i left on ur issue #7

ohmschrodinger commented:
I've made the changes as you said

- changed the pure white color to #f7f7f7
- updated the hover effect so that the pokemon-image gets the shadow effect instead of the whole card.

lord-cyclone100 commented:
Amazing work @ohmschrodinger
Keep it up!!! 
I am merging
